src/Microsoft.DiaSymReader.PortablePdb.Tests/EmbeddedSourceTests.cs

```diff
@@ -0,0 +1,194 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Roslyn.Test.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using System.IO.Compression;
+using System.Reflection.Metadata;
+using Xunit;
+
+namespace Microsoft.DiaSymReader.PortablePdb.UnitTests
+{
+    using System.Linq;
```

Comment:
SymTestHelpers copied from elsewhere, causes ctrl + . to put more usings next to it. I'll move System.Linq.